service/internal/subjectmappingbuiltin/subject_mapping_builtin_test.go

```diff
@@ -90,6 +90,31 @@ func Test_EvaluateConditionNOTINFalse(t *testing.T) {
 	assert.False(t, res)
 }
 
+// evaluate condition CONTAINS
+var containsCondition1 = policy.Condition{
+	SubjectExternalSelectorValue: ".attributes.testing[]",
+	Operator:                     policy.SubjectMappingOperatorEnum_SUBJECT_MAPPING_OPERATOR_ENUM_CONTAINS,
+	SubjectExternalValues:        []string{"option"},
```

Comment:
can we please add test cases for how multiple values in a list where only one matches are expected to be evaluated?